Source/WebKit/UIProcess/API/Cocoa/_WKWebsiteDataStoreConfiguration.mm

```diff
@@ -756,7 +756,7 @@ - (void)_setShouldAcceptInsecureCertificatesForWebSockets:(BOOL)accept
 
 - (void)setProxyConfiguration:(NSDictionary *)configuration
 {
-    _configuration->setProxyConfiguration((__bridge CFDictionaryRef)[configuration copy]);
+    _configuration->setProxyConfiguration(adoptNS([configuration copy]));
```

Comment:
Missing `.get()` here:

```suggestion
    _configuration->setProxyConfiguration(adoptNS([configuration copy]).get());
```

Reply:
Oh, but you also need to bridge cast:

```suggestion
    _configuration->setProxyConfiguration(bridge_cast(adoptNS([configuration copy]).get()));
```

Reply:
yeah, I've done that in my latest change.